core-api/src/main/java/com/optimizely/ab/config/PollingProjectConfigManager.java

```diff
@@ -1,6 +1,6 @@
 /**
  *
- *    Copyright 2019, Optimizely and contributors
+ *    Copyright 2020, Optimizely and contributors
```

Comment:
I would be 2019, 2020 or 2019-2020. copyright should cover all the years this file was modified in